doc/sphinxext/math_symbol_table.py

```diff
@@ -6,17 +6,16 @@
 symbols = [
     ["Lower-case Greek",
      6,
-     r"""\alpha \beta \gamma \chi \delta \epsilon \eta \iota \kappa
-         \lambda \mu \nu \omega \phi \pi \psi \rho \sigma \tau \theta
-         \upsilon \xi \zeta \digamma \varepsilon \varkappa \varphi
-         \varpi \varrho \varsigma \vartheta""".split()],
+     r"""\alpha \beta \gamma \delta \epsilon \zeta \eta \theta \iota \kappa \lambda \mu
+         \nu \xi \pi \rho \sigma \tau \upsilon \phi  \psi  \chi \omega \digamma
+         \varepsilon \vartheta \varkappa \varphi \varpi \varrho \varsigma""".split()],
     ["Upper-case Greek",
-     8,
-     r"""\Delta \Gamma \Lambda \Omega \Phi \Pi \Psi \Sigma \Theta
-     \Upsilon \Xi \mho \nabla""".split()],
+     6,
+     r"""\Gamma \Delta  \Theta \Lambda \Xi \Pi \Sigma \Upsilon \Phi \Psi \Omega
+      """.split()],
     ["Hebrew",
      6,
-     r"""\aleph \beth \daleth \gimel""".split()],
+     r"""\aleph \beth \gimel \daleth""".split()],
```

Comment:
```suggestion
     r"""\daleth \gimel \beth \aleph""".split()],
```

I trust @story645 's suggestion that the Hebrew should be ordered right-to-left.

Reply:
Ahh, missed that aspect of it.